src/ray/protobuf/gcs.proto

```diff
@@ -310,6 +311,11 @@ message ResourceMap {
   map<string, ResourceTableData> items = 1;
 }
 
+message StoredConfig {
```

Comment:
Not sure we need to define this type, would probably be better to just use the map directly to avoid unnecessary indirection

Reply:
The indirection is required for the storage mechanism. The template class expects the 'data' field to have RPC methods (like `SerializeAsString`).

Reply:
Gotcha, makes sense